OpenRA.Mods.Common/AudioLoaders/MP3Loader.cs

```diff
@@ -0,0 +1,60 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.IO;
+using MP3Sharp;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.Common.AudioLoaders
+{
+	public class MP3Loader : ISoundLoader
```

Comment:
I'm not sure how important this is, but so far we've only made first letters uppercase, like Aud, Wav and so on, so Mp3 would strictly speaking be more consistent with what we have.

Reply:
I tried to be consistent with the `MP3Sharp` namespace.